src/topology/algebra/valuation.lean

```diff
@@ -103,6 +101,8 @@ begin
   exact valuation.map_eq_of_sub_lt _ y_in
 end
 
+-- These instances are not inferred anymore
+
 /-- The uniform structure on a valued ring.-/
 @[priority 100]
 instance uniform_space : uniform_space R := topological_add_group.to_uniform_space R
```

Comment:
If I add this I get a lint error:
```
/- The `unused_arguments` linter reports: -/
/- UNUSED ARGUMENTS. -/
#check @valued.uniform_space /- argument 5: [_inst_3 : valued R Γ₀] (duplicate) -/
#check @valued.uniform_add_group /- argument 5: [_inst_3 : valued R Γ₀] (duplicate) -/
```

Reply:
You'll need to fiddle with the `variables` a bit. With what you had before, lean was saying "I don't see `Γ₀` in the statement, so I'm not going to use your `[valued R Γ₀]` variable that you have ready". With what you have now, there _is_ a `Γ₀` in the statement (in the `[valued R Γ₀]`!), so lean includes the `[valued R Γ₀]` variable you already have!

Perhaps the easiest fix is just to remove `[valued R Γ₀]` from `variables` and include it each time you need it.

Reply:
Yes, adding `[valued R Γ₀]` in each statement in the file instead of as a variable fixes this, I was wondering if there was another way to do this - is it possible to make Lean forget the `[valued R Γ₀]` right before `valued.uniform_space` somehow?

Reply:
I guess I could just use a section.